roles/openstack/nova/tasks/setup_nodes.yml

```diff
@@ -47,5 +47,6 @@
       ansible_ssh_host: "{{ servers.ansible_facts.openstack_servers[0].public_v4 }}"
       groups: "{{ provisioner.nodes[node_type.value.name].groups | join(',') }}"
       ansible_ssh_private_key_file: "{{ provisioner.key.file }}"
-      ansible_ssh_user: 'cloud-user'
+      ansible_ssh_user: "{{ node_type.value.ssh_user | default(provisioner.ssh_user) }}"
```

Comment:
User is too confusing in my opinion.
user for cloud? for instance? for accessing overcloud nodes?